wbooks-api/app/models/rent.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Rent < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
+
+  validates :loan, :restitution, presence: true
```

Comment:
@icoluccio should validate the presence of user and book to?

Reply:
Hm I'd say so, the only problem is if the book doesn't exist anymore but I don't think it matters much